docker/Dockerfile

```diff
@@ -61,6 +59,7 @@ RUN curl -fsSL -v -k -o ~/miniconda.sh -O https://repo.anaconda.com/miniconda/Mi
         openssl \      
         weasyprint && \
     /opt/conda/bin/conda clean -ya && \
+    pip install intel-openmp && \
```

Comment:
Why we need this change?

Reply:
Conda-forge uses `conda-forge` channel by default, which has no `intel-openmp` pkg. If we want to install `intel-openmp` through `conda`, it only can be installed through intel's conda channel by indicating `conda install intel-openmp -c intel `. I am not that we were able to use intel channel or not. I am checking this issue with related colleage.

Reply:
Yes, we can use intel channel

Reply:
Resolved